### PR TITLE
Add HelpGuidelinesURL configuration for help

### PR DESCRIPTION
**Pull request by hi-rustin**
Add `HelpGuidelinesURL` configuration for help plugin.

### CODE REVIEW

**Comment by matthyx** [comment]
@hi-rustin I think it's similar to https://github.com/kubernetes/test-infra/issues/19629 we somehow should make this global to prow instead of plugin specific...

I would make a global configuration listing all urls that should be configurable (and potentially shared between plugins) in prow installations.

Any thoughts @alvaroaleman ?

**Comment by hi-rustin** [comment]
@alvaroaleman  Could you please take a look?

**Comment by hi-rustin** [comment]
@matthyx  @alvaroaleman  ping~

**Comment by hi-rustin** [comment]
> I would make a global configuration listing all urls that should be configurable (and potentially shared between plugins) in prow installations.

I think some generic configurations can be put into the global configuration share, but such links that are only relevant to a specific plugin can be put into the plugin's own configuration.

File: prow/plugins/config.go [hunk 1]
```diff
@@ -85,6 +84,19 @@ type Configuration struct {
...
+}
+
+type Help struct {
+	// HelpGuidelinesURL is the URL of the help page, which provides guidance on how and when to use the help wanted and good first issue labels.
+	// The default value is "https://git.k8s.io/community/contributors/guide/help-wanted.md".
+	HelpGuidelinesURL string `json:"help_guidelines_url,omitempty"`
+}
+
+func (h *Help) SetDefaults() {
+	if len(h.HelpGuidelinesURL) == 0 {
```
Comment (fejta) [inline prow/plugins/config.go:97]: `if h.HelpGuidelinesURL == "" {`

File: prow/plugins/config.go [hunk 2]
```diff
@@ -85,6 +84,19 @@ type Configuration struct {
...
+	Help                 Help                         `json:"help,omitempty"`
+}
+
+type Help struct {
+	// HelpGuidelinesURL is the URL of the help page, which provides guidance on how and when to use the help wanted and good first issue labels.
+	// The default value is "https://git.k8s.io/community/contributors/guide/help-wanted.md".
+	HelpGuidelinesURL string `json:"help_guidelines_url,omitempty"`
+}
+
+func (h *Help) SetDefaults() {
```
Comment (fejta) [inline prow/plugins/config.go:96]: Who do we need to export this to?

**Review by fejta — APPROVED** [review]
/hold

thanks!

File: prow/plugins/config.go (hunk 1 above)
Comment (hi-rustin) [reply prow/plugins/config.go]: Fixed.

File: prow/plugins/config.go (hunk 2 above)
Comment (hi-rustin) [reply prow/plugins/config.go]: Fixed.

**Pull request merged**